src/composables/ViewRoutingSlip/useRoutingSlipTransaction.ts

```diff
@@ -0,0 +1,28 @@
+import { ref, watch } from '@vue/composition-api'
+
+import { Invoice } from '@/models/Invoice'
+import { createNamespacedHelpers } from 'vuex-composition-helpers'
+
+const routingSlipModule = createNamespacedHelpers('routingSlip') // specific module name
+const { useState } = routingSlipModule
+
+// Composable function to inject Props, options and values to RoutingSlipTransaction component
+export default function useRoutingSlipTransaction (_, context) {
+  // vuex state
+  const { routingSlip } = useState(['routingSlip'])
+  const invoices = ref<Invoice[]>([])
```

Comment:
can we use invoices inside useTransactionDataTable composition api itself. since we are not using this dataexcept passing

Reply:
I thought of not coupling it with store. But yeah, we are use two watches which can be optimized.